src/jit-analyze/jit-analyze.cs

```diff
@@ -125,14 +138,14 @@ public class MethodInfo
         {
             public string name;
             public int totalBytes;
-            public int prologBytes;
+            public double perfScore;
```

Comment:
I don't remember seeing prologBytes before in the output, but are you sure you want to remove it, instead of just adding perfScore as an additional field?

Reply:
It was never used